src/Spectre.AngularClient/src/app/divik-results/divik/divik.component.html

```diff
@@ -3,9 +3,19 @@
   {{configDescription}}
 </div>
 
+<md-form-field *ngIf="divikConfig">
+  <input mdInput
+         type='number'
+         min="0"
+         max={{divikConfig.Level}}
+         (change)="changeLevel($event.target.value)"/>
+  <md-hint align="center">Max value: divikConfig.Level</md-hint>
```

Comment:
This is not necessarily true. If the experiment has been ran with `useLevels: false`, backend should estimate tree depth and exhibit it to the frontend.

Reply:
So if we haven't had this information yet it could be another task